stylelint-polaris/plugins/coverage/index.js

```diff
@@ -55,10 +55,79 @@ module.exports = stylelint.createPlugin(
           },
         );
       }
+
+      const disabledCoverageWarnings =
+        result.stylelint.disabledWarnings?.filter((disabledWarning) =>
+          disabledWarning.rule.startsWith(ruleName),
+        );
+
+      if (!disabledCoverageWarnings?.length) return;
+
+      const disabledCoverageLines = Array.from(
+        new Set(disabledCoverageWarnings.map(({line}) => line)),
+      );
+
+      // Ensure all stylelint-polaris/coverage disable comments
+      // have a description prefixed with "polaris:"
+      for (const disabledRange of result.stylelint.disabledRanges.all) {
+        if (
+          !isDisabledCoverageRule(disabledCoverageLines, disabledRange) ||
+          disabledRange.description?.startsWith('polaris:')
```

Comment:
Love this. I do want to make sure we are aligned with the `polaris:` prefix for disabling Polaris rules. It could be `p-ignore` `p-skip` `p-context` lots of different things. The main thing is that it is searchable and the value we want for the long term.

Not against `polaris` but might be worthwhile discussing before shipping.

Reply:
Great points. We could change it if needed in a future major and have overlap of multiply supported prefixes. So maybe we start with `polaris:` for simplicity then can introduce a convention for others later on.

Just some alternatives prefixes:
- `yo:`
- `give-me-the-context:`
- `but-why:`
- `promise-i-need-this:`

Reply:
Yes, great callout!  My main focus in this iteration was ensuring we can capture disabled coverage rules and validate some form of context was provided for tracking. I landed on `polaris:` to avoid redundancy as the alternatives (`stylelint-polaris:`, `p-ignore`, `p-skip`, etc.) are already communicated by the `stylelint-disable` comment. That said, I'm happy to make any changes to the required prefix 👍